concourse/steps/component_descriptor.mako

```diff
@@ -113,6 +113,9 @@ v2_outfile = os.path.join(
   component_descriptor_fname(schema_version=gci.componentmodel.SchemaVersion.V2),
 )
 
+ctf_out_dir = os.path.join(descriptor_out_dir, 'ctf')
```

Comment:
how about using a more expressive name for the directory? nobody will have to type it (except for you in this particular codeline^^)..

how about sth like `cnudie_transport_format-out.d`?